ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```
@@ -1552,6 +1552,26 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-publish-mixed-ingress-internal
+- as: baremetal-compact-agent-ipv4-static-day2-f14
```

```suggestion
- as: baremetal-compact-agent-ipv4-static-day2-amd-mixarch-f14
```
we use amd-mixarch to stand for the cluster is an amd cluster with the arm node scale up